### PR TITLE
refactor: 8268 updated the architecture

### PR DESCRIPTION
**Pull request by sharonrussell**
#### 📲 What

Updated the architecture to be more readable, understandable and contain clearer links to Martin Fowlers DDD overview 

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: docs/workloads/azure/backend/netcore/architecture/architecture_overview_netcore.md [hunk 1]
```diff
@@ -18,101 +18,52 @@ keywords:
   - stacks
 ---
 
+### Domain-Driven Design (DDD) Overview
 
-## DDD Architecture Overview
+For a comprehensive understanding of Domain-Driven Design, refer to Martin Fowler's notes on [Domain-Driven Design](https://martinfowler.com/bliki/DomainDrivenDesign.html).
 
-![Solution Diagram](/img/cqrs_diagram.png)
```
Comment (alexanderwjrussell) [inline docs/workloads/azure/backend/netcore/architecture/architecture_overview_netcore.md:24]: Was the diagram not adding anything?

Comment (sharonrussell) [reply docs/workloads/azure/backend/netcore/architecture/architecture_overview_netcore.md:24]: I believe it is better suited in a section about the solution structure and not a general information section on DDD

**Review by alexanderwjrussell — APPROVED** [review]

**Pull request merged**